freqtrade/optimize/hyperopt_loss_sortino.py

```diff
@@ -39,7 +39,7 @@ def hyperopt_loss_function(results: DataFrame, trade_count: int,
         results.loc[total_profit < 0, 'downside_returns'] = results['profit_percent']
         down_stdev = np.std(results['downside_returns'])
 
-        if np.std(total_profit) != 0.0:
+        if down_stdev != 0:
```

Comment:
are you sure about changing `0.0` to `0`?

Reply:
should not be a problem, it's a constant and will be cast to python if needed comparing float with it

Reply:
this should call 
```
(down_stdev).__neq__(0)
```
i.e. call np.float64.__neq__, which knows how to handle ints...

Reply:
if I'm not wrong here, though...